config/main.py

```diff
@@ -394,6 +394,18 @@ def _get_platform():
                 return tokens[1].strip()
     return ''
 
+def _get_sonic_generated_services():
+    if not os.path.isfile('/etc/sonic/generated_services.conf'):
+        return None
+    generated_services_list = []
+    with open('/etc/sonic/generated_services.conf') as generated_service_file:
```

Comment:
`'/etc/sonic/generated_services.conf'` is used multiple times. Suggest to define it as a constant at the top of this function. It can be moved to the top of the file in the future if it is needed elsewhere.

Reply:
@jleveque  Done